### DISCUSSION TITLE
for controlling Krake

### DISCUSSION THREAD

**Discussion by nk25719** [PubInv/krake#46] [Q&A]
What do you prefer for controlling devices: a rotary encoder knob or traditional switches? Why do you find one more convenient or effective than the other?

**Comment by RobertLRead** [comment]
Rotary Encoders.  Mainly because: Dr. Erich Shulz, an
obstetric anesthesiologist, says that is the best hardware UI when you are
wearing gloves (such as latex medical gloves), and when you are under
stress.)  With a single device, you open up a large number of input options
(IF you have a display to integrate with, which we do.)

On Fri, Oct 18, 2024 at 1:08 PM nagham.kheir25 ***@***.***>
wrote:

> What do you prefer for controlling devices: a rotary encoder knob or
> traditional switches? Why do you find one more convenient or effective than
> the other?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PubInv/krake/discussions/46>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABINEH66CXOUZ6FG4SPHYPTZ4FFCZAVCNFSM6AAAAABQGOKXCOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGMZTQNJVHA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Robert L. Read, PhD
Twitter: @RobertLeeRead @pubinvention
Public Invention: https://www.pubinv.org
Join Our Mailing list: ***@***.***
YouTube: https://www.youtube.com/channel/UCJQg_dkDY3KTP1ybugYwReg
Medium: ***@***.***

**Comment by ForrestErickson** [comment]
There is a LOT to define once we have the opportunity for a 20x4 LCD display (with the rotary encoder) to navigate features through menus.
The LCD and Rotary encoder hardware interface is the TRIVIAL easy part

Here is an example of the hardware with a very very very simple user interface (No menus to navigate)
https://wokwi.com/projects/390353236396127233

**Comment by ForrestErickson** [comment]
A five button UI is an alternative too.
![image](https://github.com/user-attachments/assets/38a9a9ce-c164-4dca-a2cc-c8399d8f55e9)
